pkg/pillar/cmd/zedmanager/handleclusterapp.go

```diff
@@ -1,6 +1,3 @@
-// Copyright (c) 2024-2025 Zededa, Inc.
```

Comment:
I think copyright note wasn't suppose to be removed here... just bring it back...

Reply:
ok